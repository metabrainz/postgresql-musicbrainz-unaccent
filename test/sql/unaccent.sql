CREATE EXTENSION musicbrainz_unaccent;

SELECT musicbrainz_unaccent('ľščťžýáí');
SELECT musicbrainz_unaccent('foo—bar‒baz');
SELECT musicbrainz_unaccent('nonunicode');
SELECT musicbrainz_unaccent('');
SELECT musicbrainz_unaccent(null);
SELECT musicbrainz_unaccent(repeat('ä', 65536)) = repeat('a', 65536);

SELECT ts_lexize('musicbrainz_unaccentdict', 'ľščťžýáí foo—bar‒baz nonunicode');
