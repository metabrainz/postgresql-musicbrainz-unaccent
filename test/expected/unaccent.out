CREATE EXTENSION musicbrainz_unaccent;
SELECT musicbrainz_unaccent('ľščťžýáí');
 musicbrainz_unaccent 
----------------------
 lsctzyai
(1 row)

SELECT musicbrainz_unaccent('foo—bar‒baz');
 musicbrainz_unaccent 
----------------------
 foo-bar-baz
(1 row)

SELECT musicbrainz_unaccent('nonunicode');
 musicbrainz_unaccent 
----------------------
 nonunicode
(1 row)

SELECT musicbrainz_unaccent('');
 musicbrainz_unaccent 
----------------------
 
(1 row)

SELECT musicbrainz_unaccent(null);
 musicbrainz_unaccent 
----------------------
 
(1 row)

SELECT musicbrainz_unaccent(repeat('ä', 65536)) = repeat('a', 65536);
 ?column? 
----------
 t
(1 row)

SELECT ts_lexize('musicbrainz_unaccentdict', 'ľščťžýáí foo—bar‒baz nonunicode');
              ts_lexize              
-------------------------------------
 {"lsctzyai foo-bar-baz nonunicode"}
(1 row)

